### DISCUSSION TITLE
Card changes where to check

### DISCUSSION THREAD

**Discussion by DERTAZ** [bafolts/terraforming-mars#5234] [General]
Since January 6, 3 corporation cards and a project card have been nerfed:
Ancient Shipyards
Darkside of the Moon Syndicate
Luna First Incorporated
Luna Trade Federation
Where can I check what changes have been made or what the card looked like before the changes?
Is there any way to restore the old version of the card?

**Comment by kberg** [comment]
https://github.com/terraforming-mars/terraforming-mars/wiki/The-Moon#differences-from-the-designers-original-intent describes which cards have changed. I do not include any of the details, because the existing behavior is kind of what matters. If you know how to change the code, you can look at the history of the card and make your own server with the original behaviors.
